tests/RazorLight.Tests/RazorLight.Tests.csproj

```diff
@@ -58,6 +52,12 @@
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.1.5" />
   </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.0" />
```

Comment:
We should use a higher version of Microsoft.NET.Test.Sdk - are you using an out of date version of Visual Studio?  Otherwise I am a bit surprised the nuget package manager would add 16.0.0 unless you added it manually.  Is there some reason to specify 16.0.0?  (Genuinely curious if you know something I don't or if it was just a typo.)